### DISCUSSION TITLE
Admin interface?

### DISCUSSION THREAD

**Discussion by MACscr** [plausible/analytics#667] [Self-Hosted Support] [answered]
Thanks for making this awesome GA alternative and with a nice clean UI. I was able to successfully install it and get it running and i setup the default admin using the env file. But, i have no idea what the url is for the admin backend and when i try to use those details on the root page of the site, it just lets me create a new account as if i was just a regular user. Am i missing something basic that I am not seeing in the docs?

**Comment by ukutaht** [comment, marked as answer]
We don't really have an admin interface yet, sorry.. the self-hosted installation allows people to create normal accounts pretty much and that's it.

**Comment by MACscr** [comment]
Anyway to disable registration while creating accounts manually through an api? I guess then though there would be no way for an admin to view any of the data, etc. Trying to figure out a solution for my freelance clients. Not interested in selling a solution, etc. 

- Mark

**Reply by ukutaht** [reply]
Once we implement it the best way would be to create a team with roles  #176

Currently you have 2 options:
1. Create all the sites from your account and send to your clients a shared link for each website.
2. Allow clients to register their own account. In this case you would not have 'god mode' to see all the stats

Sorry I know we're lacking severely in this area at the moment. We will improve these things later this year once I've dealt with some other priorities

**Comment by MACscr** [comment]
Im a bit confused though on your /hosting and /analytics repos and then with your merges like https://github.com/plausible/analytics/pull/557. Does the the "self" hosted version lag behind and if we are more adventurous, we can piece together the different projects for a more full featured solution?

**Reply by metmarkosaric** [reply]
#557 has nothing to do with the self-hosted version. 

We release the self-hosted version twice per year - it's a long term release and you can read more about the schedule here: https://plausible.io/blog/building-open-source.

The only releases/merges you need to care about as a self-hoster are those ones listed here: https://github.com/plausible/analytics/releases

**Reply by MACscr** [reply]
Right, but it does have to do with code that you are publicly making available and AGPL licensed, so I am free to use it, correct? As mentioned, I am not looking to start a competing service and I am not even against subscribing to help financially. The self hosted edition is just extremely limited in its features and and your merges show that you have already created additional functionality in your main repo and in your master branch (not a dev or other branch). If it was simply a repo for internal use, then I don't see why it would be public vs private. You have already stated on your sites that your self-hosted edition is not supported, so I guess I dont see the difference in trying out your main repo as well. I hope I am not coming off as abrasive as that I not my intent as at all and I appreciate all that your doing here.

**Reply by metmarkosaric** [reply]
Self hosted version is the same as the hosted version. There are no differences. You can check in the link that I shared.

**Comment by ukutaht** [comment]
> Im a bit confused though on your /hosting and /analytics repos and then with your merges like #557. Does the the "self" hosted version lag behind and if we are more adventurous, we can piece together the different projects for a more full featured solution?

Ah yes. The self-hosted version does lag behind but it should actually include this merge.

Look, there are things in the codebase that we are running as an experiment or we are still just working on. This is one of them. I am also experimenting with different error loggers. We haven't documented how to use the CRM because it's not terribly useful or full-featured at the moment. It's just for us to see whether it's better to manage things than what I'm doing now which is issuing SQL commands manually.

In the future we could throw this away, start a compeltely different approach, or maybe document this area and encourage people to use it. We don't know yet, as you can see the merge is about a month old and we are still evaluating Kaffy internally.

The code is in the public domain, do with it what you wish (except sell a derived, closed-source work based on it). Sorry I will not document here how to use the CRM for the reasons mentioned. We also cannot guarantee that it will be part of the next release.

**Reply by rdvo** [reply]
is there a way to see stats for ALL domains on everyones account? or do i let ppl register thne just share them to all the sites on my mai nadmin account?
